File: YouTrackReportApp/Models/Youtrack/ReportModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using YouTrackReports.Models.Youtrack;

namespace YouTrackReports.Models
{
    public class ReportModel
    {
        public int ReportId { get; set; }
        public Summary SummaryInformation { get; set; }
        public IndividualEmploymentTable TableDataInformation { get; set; }
        public ReportModel()
        {
            this.SummaryInformation = new Summary();
            this.TableDataInformation = new IndividualEmploymentTable();
        }
        public void Initialize(SummaryModel summary, IndividualEmploymentTable tableInformation, int reportId = 0)
        {
            this.SummaryInformation.SummaryModel = summary;
        }
    }








}

File: YouTrackReportApp/Components/Models/ReportModel.ts
export class ReportModel
{
    summaryInformation?: Summary;
    tableDataInformation?: IndividualEmploymentTable;
}

export class Summary
{
    summaryModel: SummaryModel;
}

export class IndividualEmploymentTable
{
    individualEmploymentModel: IndividualEmploymentModel[]
}

export class SummaryModel
{
    sumPlanningMark: number;
    sumActualMark: number;
    developersCount: number;
    averageParticipationDegree: number;
}

export class IndividualEmploymentModel
{
    id: number;
    developer: string;
    scopeOfWork: number;
    participationDegree: number;
}